perception/sub8_perception/nodes/path_marker_finder.py

```diff
@@ -35,10 +35,18 @@ class PathMarkerFinder():
     TODO: Implement Kalman filter to smooth pose estimate / eliminate outliers
     """
     # Model of four corners of path marker, centered around 0 in meters
-    PATH_MARKER = np.array([[0.6096,  -0.0762, 0],
-                            [0.6096,  0.0762,  0],
-                            [-0.6096,  0.0762, 0],
-                            [-0.6096, -0.0762, 0]], dtype=np.float)
+    LENGTH = 0.6096 # Longer side of path marker in meters
+    WIDTH = 0.0762 # Shorter Side
+    PATH_MARKER = np.array([[LENGTH,  -WIDTH, 0],
+                            [LENGTH,  WIDTH,  0],
+                            [-LENGTH,  WIDTH, 0],
+                            [-LENGTH, -WIDTH, 0]], dtype=np.float)
+
+    # Scale model of marker as a cv2 contour, for use in cv2.matchShape
+    PATH_MARKER_2D = np.array([[[0, 0]],
+                               [[WIDTH*10000, 0]],
+                               [[WIDTH*10000, LENGTH*10000]],
```

Comment:
They're going to have to be ints for the cv2 call to work, so get more resolution by multiplying by 1000 for rounding reasons.